src/backend/distributed/Makefile

```diff
@@ -36,9 +36,37 @@ OBJS += \
 
 all: cdc
 
+setup_cdc_build_dir_copy_files:
```

Comment:
I feel like it's a bit confusing to have all the logic here rather than in the CDC makefile, but maybe just add some comments on what we're doing.

Reply:
Moved the CDC decoder build setup and running to CDC makefiles.